processor/processor.go

```diff
@@ -68,19 +70,44 @@ func (p *processor) Process(d delivery.Delivery) error {
 }
 
 func (p *processor) run() int {
+	start := time.Now()
 	p.log.Info("Processing message...")
-	defer p.log.Info("Processed!")
+	defer func() {
+		p.log.Info(fmt.Sprintf("Processed (%s)!", time.Since(start)))
+	}()
 
 	var out []byte
 	var err error
 	capture := p.cmd.Stdout == nil && p.cmd.Stderr == nil
 
+	ticker := time.NewTicker(10 * time.Second)
+	tickerCtx, tickerCtxCancelFunc := context.WithCancel(context.Background())
+
+	go func() {
+		for {
+			select {
+			case <-ticker.C:
+				durationSinceStart := time.Since(start)
+				if durationSinceStart > 5*time.Minute {
```

Comment:
Pure pinaillage mais le sortir du if et le mettre au dessus du for en constante ce serait plus joli

Reply:
changé :ok_hand: